homework-1/src/templates/slider.html

```diff
@@ -0,0 +1,90 @@
+<div class="slider">   
+   <div class="slider__area-logo">
+      <a id="sliderLogo" class="slider__logo-link"><img src="/assets/images/SkoutGlossary.png" alt="" class="slider__logo" ></a>
+   </div>
+   <div class="slider__section">
+            <div class="slider__left">
+                <div class="slider__left-one" id="sliderLeftOne">
+                   <div class="slider__dots">
+                       <div class="hover-dot"></div><p class="slider__left-text-link" id="dotOne">Botnet</p>
+                    </div>
+                   <div class="slider__dots">
+                       <div  class="hover-dot"></div><p class="slider__left-text-link" id="dotTwo">Breach</p>
```

Comment:
`slider__left-text-link` - if it is a link, use `<a>` with href attribute. Get rid of id's. You can use href value to navigate between slides.